src/webextension/locales/en-US/widgets.ftl

```diff
@@ -1,10 +1,13 @@
+copy-to-clipboard-button = Copy
+copy-to-clipboard-copied = ✔ Copied
```

Comment:
Just explicitly noting what I hope is obvious: how this unicode renders depends on the font in use.  If we change the font, it can (read: likely will) change how this renders.

It will still be a checkmark (I hope), but the shape can be different in noticeable ways.

I am perfectly happy to use unicode instead of images wherever possible.

Reply:
Yeah, we *should* know what all the fonts are (and Firefox is pretty good about falling back to another font if the current one doesn't have that glyph), so it's not a complete unknown. Still, I'll leave it up to @changecourse to decide what we should do here. Using Unicode does have the benefit of being a fair bit simpler though...